youtube_dl/extractor/instagram.py

```diff
@@ -356,6 +356,24 @@ def _real_extract(self, url):
             self._extract_graphql(data, url), user_or_tag, user_or_tag)
 
 
+class InstagramSavedIE(InstagramPlaylistIE):
+    _VALID_URL = r'https?://(?:www\.)?instagram\.com/(?P<id>[^/]{2,})/saved/?(?:$|[?#])'
+    IE_DESC = 'Instagram saved media'
+    IE_NAME = 'instagram:saved'
+
+    _QUERY_HASH = '8c86fed24fa03a8a2eea2a70a80c7b6b'
+
```

Comment:
Add a test.

Reply:
This test requires cookies and is specific to the user. Anyone who wants to test this must change the URL and add `params['cookiefile'] = ...` in `test/test_download.py`.